src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,51 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String CSV_SEPARATOR = ",";
 
-    public void getStatistic(String fromFileName, String toFileName) {
+    private String getResult(int[] cases) {
+        return "supply" + CSV_SEPARATOR + cases[0] + System.lineSeparator()
+                + "buy" + CSV_SEPARATOR + cases[1] + System.lineSeparator()
+                + "result" + CSV_SEPARATOR + (cases[0] - cases[1]);
+    }
+
+    public int[] setStatistic(String fromFileName) {
```

Comment:
Did you see this comment??? and link?
> should all methods be public?
https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents